pkgs/development/python-modules/free-proxy/default.nix

```diff
@@ -13,15 +14,16 @@ buildPythonPackage rec {
   pyproject = true;
 
   src = fetchFromGitHub {
-    owner = "jundymerk";
+    owner = "jundymek";
     repo = "free-proxy";
-    rev = "refs/tags/v${version}";
-    hash = "sha256-1hTOMbsL1089/yPZbAIs5OgjtEzCBlFv2hGi+u4hV/k=";
+    rev = "6faf092b0bd66b5004dc2dd67f8272c311c61bbb";
```

Comment:
It's just nurl. You don't have to

Reply:
all neccessary changes made. everyhing works as normal now.